source/open-definition-2.1-dev.markdown

```diff
@@ -42,8 +42,7 @@ accompany the **work**.
 
 The **work** *must* be provided in an open format. An open format is
 one which places no restrictions, monetary or otherwise, upon its use and can be fully processed
-with at least one free/libre/open-source software tool. Data *must* be machine-readable and 
-*should* be provided in bulk.
+with at least one free/libre/open-source software tool. The **work** *should* be provided in the form preferred for making modifications to it.
```

Comment:
I like the changes. I'm not sure about "preferred". I think we would either:
- need to separately provide some guidance or examples on preferred formats as peoples preferences will vary.
- change "preferred" to something that drives people to at least 3 star open data publishing.